src/app/home/sagas.ts

```diff
@@ -0,0 +1,18 @@
+import { SagaIterator } from 'redux-saga';
+import { call, takeLatest } from 'redux-saga/effects';
+
+import { client } from '../../graphql/api';
+
+import { CHANGE_PASSWORD } from './actions';
+
+export function* changePasswordSaga(action: ReturnType<typeof CHANGE_PASSWORD.TRIGGER>): SagaIterator {
+    try {
+        yield call(client.changePassword, action.payload.currentPass, action.payload.newPass);
+    } catch (err) {
+        throw new Error(err);
```

Comment:
WTF, you throwing exceptions in catch, seriously? We should call FlashBar here and on "try" after success